app/metrics.yaml

```diff
@@ -1984,3 +1984,57 @@ addons:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+
+startup.timeline:
+  framework_start:
+    send_in_pings:
+      - startup-timeline
+    type: timespan
+    time_unit: nanosecond
+    description: >
+      The duration the Android framework takes to start before letting us run code in
+      `*Application.init`. This is calculated from `appInitTimestamp - processStartTimestamp`.
+      `processStartTimestamp` is derived from the clock tick time unit, which is expected to be
+      less granular than nanoseconds. Therefore, we convert and round our timestamps to clock ticks
+      before computing the difference and convert back to nanoseconds to report.
+
+      For debugging purposes, `clock_ticks_per_second`, which may vary between devices, is also
+      reported as a metric.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/8803
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/9788#issuecomment-610648980
```

Comment:
Note that you need to link to the **answers** of the data-review by the data stewards.

Reply:
I'll apply these changes after data review. :)